NachoSpigot-API/src/main/java/org/bukkit/event/player/PlayerLocaleChangeEvent.java

```diff
@@ -4,35 +4,26 @@
 import org.bukkit.event.HandlerList;
 
 /**
- * Called when the locale of the player is changed.
+ * Called when a player changes their locale in the client settings.
  */
 public class PlayerLocaleChangeEvent extends PlayerEvent {
+
     private static final HandlerList handlers = new HandlerList();
-    private final String oldLocale;
-    private final String newLocale;
 
-    public PlayerLocaleChangeEvent(final Player player, final String oldLocale, final String newLocale) {
-        super(player);
-        this.oldLocale = oldLocale;
-        this.newLocale = newLocale;
-    }
+    private final String locale;
 
-    /**
-     * Gets the locale the player switched from.
-     *
-     * @return player's old locale
-     */
-    public String getOldLocale() {
```

Comment:
One more thing, please add these methods back and just return the `locale` variable for backwards compatibility.

Reply:
I don’t think it’s necessary. This event was implemented during a time when not a lot of people used Paper and plugin developers wouldn’t use it’s API and in 1.9, the package for this event would be moved to Paper’s new package.